src/sortStudents.ts

```diff
@@ -1,16 +1,53 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
 // create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function averageGrades(grades: number[]): number {
```

Comment:
```suggestion
export function getAverageGrades(grades: number[]): number {
```
https://www.30secondsofcode.org/articles/s/javascript-naming-conventions#functions